lib/Feature/XmlConstants.h

```diff
@@ -0,0 +1,81 @@
+#ifndef VARA_FEATURE_XMLCONSTANTS_H
+#define VARA_FEATURE_XMLCONSTANTS_H
+
+#include "libxml/tree.h"
+
+namespace vara::feature {
+
+class XmlConstants {
+public:
+  XmlConstants() = delete;
```

Comment:
```suggestion
  XmlConstants() = delete;
  XmlConstants(const XmlConstants &) = delete;
  XmlConstants &operator=(const XmlConstants &) = delete;
  XmlConstants(XmlConstants &&) noexcept = delete;
  XmlConstants &operator=(XmlConstants &&) noexcept = delete;
```
We should also specify the other special member functions even if we don't use them